views/teams/team.hbs

```diff
@@ -231,11 +231,11 @@
 {{#content "body"}}
 
 <div class="modal-footer">
-    <button type="button" class="btn btn-secondary btn-close">
+    <button type="button" class="btn btn-secondary btn-close" data-testid="btn-abort-action">
         {{$t "global.button.cancel"}}
     </button>
 
-    <button type="submit" class="btn btn-primary btn-submit">
+    <button type="submit" class="btn btn-primary btn-submit" data-tesid="btn-submit-action">
```

Comment:
You have a small typo here: data-tesid="btn-submit-action" <- the test is missing a t

Reply:
fixed